templates/index.html

```diff
@@ -49,25 +49,24 @@ <h2>The app store for Linux</h2>
     <h3 class="p-muted-heading">Official snaps from major publishers</h3>
   </div>
   <div class="row">
-    <div class="p-fluid-grid--centered">
-      <span class="p-logo-image p-fluid-grid__item--small u-align--center">
-          <a href="/firefox"><img src="https://assets.ubuntu.com/v1/aae93857-logo-mozilla--snapcraft-homepage.svg" alt="Mozilla" /></a>
-      </span>
-      <span class="p-logo-image p-fluid-grid__item--small u-align--center">
+    <div class="col-small-2 col-medium-3 col-2 col-no-scale u-align--center">
```

Comment:
I think it was suggested we could try using inline images pattern here.
I gave it a try (to try avoid adding customized grid classes) and it kinda works (but not without custom styling):
https://jsfiddle.net/bartaz/oyx83meu/

So, it's one custom modification of a pattern against the other. Not sure which one I prefer.

What I don't like about the col-no-scale col-no-scale--pushed is that it ties a lot into how many logos there are. If we would add a new logo to the list, we need to figure out sizing and when images should drop to new line again.
On the other hand... if we add more images we probably would need some design/style changes anyway...